### PR TITLE
2021 performance improvements

### PR DESCRIPTION
**Pull request by zarino**
Performance improvements as part of https://github.com/mysociety/theyworkforyou/issues/1421.

Most of this will probably be imperceptible, but it’s nice to tidy stuff up.

Ripped out a bunch of JavaScript plugins, moved all of the remaining JavaScript to the end of the body for a slight performance boost, added cachebusting query params where they were missing, and removed some IE8 workarounds.

I’d hoped we could drop either the 600 or 700 weight of Source Sans Pro, but they’re both fairly important for different things :-/

The only user-facing change is that scrolling down the MP pages will no longer highlight the "current" link in the sidebar. We could write something to acheive this with IntersectionObserver, but it feels like a lot of faff, and I’m not sure it’s worth it. Any thoughts @dracos ?

### CODE REVIEW

File: www/includes/easyparliament/templates/html/mp/profile.php [hunk 1]
```diff
@@ -66,8 +65,8 @@
                 <?php if ( !$current_member[HOUSE_TYPE_COMMONS] ) { ?>
                     <?php if (count($policyPositions->positions) > 0) { ?>
                     <div class="panel">
-                        <a name="votes"></a>
-                        <h2 data-magellan-destination="votes">A selection of <?= $full_name ?>'s votes</h2>
+                        <a name="votes" id="votes"></a>
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/mp/profile.php:68]: Why have both name and ID on these?

File: www/includes/easyparliament/templates/html/mp/profile.php [hunk 2]
```diff
@@ -66,8 +65,8 @@
...
+                        <h2>A selection of <?= $full_name ?>'s votes</h2>
```
Comment (dracos) [inline www/includes/easyparliament/templates/html/mp/profile.php:69]: curly quote to match others?

**Review by dracos — CHANGES_REQUESTED** [review]
Looks good! A few small changes is all.

File: bin/make-css [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+for argument in "$@"
+do
+    [[ "$argument" == "--watch" ]] && MODE="watch"
+done
+
+export PATH="$DIR/vendor/bundle-bin:$PATH"
+
+cd "$DIR/www/docs/style"
+compass "$MODE"
```
[the remark this answers is not in the document]
Comment (zarino) [reply bin/make-css:16]: Ah, I was just copying what `deploy.bash` did. Maybe they should both use `bundle exec compass`?

File: www/includes/easyparliament/templates/html/mp/profile.php (hunk 1 above)
Comment (zarino) [reply www/includes/easyparliament/templates/html/mp/profile.php:68]: Oops, yes, not sure why I did this. Jumping to elements by `name` attribute works fine, without an `id`. Force of habit.

File: bin/make-css (hunk 3 above)
[the remark this answers is not in the document]
Comment (zarino) [reply bin/make-css]: I’ve gone ahead and added `bundle exec` to both anyway.

**Review by dracos — APPROVED** [review]
Can we get a webpagetest or two before to compare to after? :)

**Pull request merged**